### ISSUE TITLE
Where are the download instructions!? (Not working on latest updated Windows 10)

### ISSUE THREAD

**Issue by dannYves** [silnrsi/graphite#19]
There is no documentation anywhere on how to correctly compile this package!! I did see somewhere where cmake is needed, but that this did not work. Instead Microsoft Visual Studios 2008 is used but I don't understand the instructions.

I tried to extract the original graphite compiler but it is not working either and continues to give me an error where a dll file can not be found ...and with no other proper installation instructions it brings me here to ask for assistance.

**Comment by mhosken** [comment]
did you look at doc/building.txt? Our hope is that that document will meet the need.

**Comment by dannYves** [comment]
> @mhosken 
did you look at doc/building.txt? Our hope is that that document will meet the need.
I found it after searching all over right after I commented but I was unable to generate anything, I am not very knowledgeable on scripting so it was difficult for me to entirely understand. I ultimately went on this site: http://www.readytext.co.uk/?p=2903, and followed the directions the best way I could but in the command prompt I kept getting an error that said: "cmake is unrecognizable as a command", so I inputted set PATH="C:\Program Files\CMake\bin\";%PATH% per this site: https://stackoverflow.com/questions/19176029/cmake-is-not-recognised-as-an-internal-or-external-command, and the path finally worked but -- 

now I am getting  another error- here's a copy of the entire command: C:\Users\Yandiel>cmake -G "Visual Studio 2017" -DCMAKE_BUILD_TYPE=Release -DGRAPHITE2_COMPARE_RENDERER:BOOL=OFF
CMake Error: Could not create named generator Visual Studio 2017

I couldn't really find any information on the internet.

EDIT:
In  reference to the 'building notepad'... in step #1 I don't know how to create build directory, in step #3 I don't know how to generate files for a build system, and #3 I know nothing about binaries, and in step #4 since I was unable to complete the prior steps I am far from being able to clean the project. And surprisingly I have not been able to find any tutorials or information for any of these procedures, which is a shock to me because not everyone is knowledgeable in every aspect or application in programming script.

**Comment by mhosken** [comment]
It sounds like this is hard going for you. I am wondering why you are wanting to build the graphite engine in the first place. Let's ensure that after all this work you are going to get what you expect. Building the graphite engine results in a dll that can only be used by an application that expects the graphite dll. It won't automatically cause an application that doesn't support Graphite to start supporting it.

**Comment by dannYves** [comment]
rtf (right to left) directionality and stacking diactricts which are not supported in most font creators that I'm aware of.

And yes I am having a very hard time since programming isn't my strongest suit lol. I've just always been a hands on person jack of all trades where I try to get everything done myself.

**Comment by mhosken** [comment]
You don't need to build the graphite engine to create a graphite font. The only reason to build the graphite engine is to integrate it into an application you are building. Instead I recommend using graide, the graphite ide, specially designed for those wanting to add graphite capability to their fonts. See here: http://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graide Then, once you have a font, you can use it in various applications including LibreOffice and Firefox.

**Comment by dannYves** [comment]
Thank you for your help mhosken.. I did a little more research after reading your reply...

I must have misunderstood something along the line while creating my font because it's my understanding that the Graphite engine is used along with GDL files to extract a new font package... I was unaware that the graphite engine was a.dll...

I did end up downloading graphite Graide and I was toying around with it, it's much more user friendly than what I've been dealing with though there's an error that comes up when I do the tutorial which says that one of the GDL scripted codes is "unreadable" and it will not expose the cap lock letters like it shows in the pdf..I attempted it three times and all times it gave me the same error. Anyway I could still play around with it until I understand it more.. I really do feel like the graphite Graide application could go places, there's many people who could take advantage of it if it's put out there.

As far as the graphite engine goes, you're saying I would'nt need it?
